frontstage_api/resources/surveys/generate_eq_url.py

```diff
@@ -48,4 +54,4 @@ def get():
                                         category=category,
                                         description=f"Instrument {case['collectionInstrumentId']} launched by {party_id} for case {case_id}")
 
-        return {"eq_url": eq_url}
+        return make_response(jsonify(eq_url), 200)
```

Comment:
Should this be jsonify(eq_url= eq_url)? Might want a test to check it's not going to break the API if there isn't already